dropwizard-validation/src/test/java/io/dropwizard/validation/ConstraintViolationsTest.java

```diff
@@ -0,0 +1,108 @@
+package io.dropwizard.validation;
+
+import io.dropwizard.validation.valuehandling.OptionalValidatedValueUnwrapper;
+import org.hibernate.validator.HibernateValidator;
+import org.junit.Test;
+
+import javax.validation.*;
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.executable.ExecutableValidator;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SuppressWarnings({"UnusedDeclaration"})
+public class ConstraintViolationsTest {
+
+    private final Validator validator = Validation
```

Comment:
Could we use a shorten version `Validation.buildDefaultValidatorFactory().getValidator()`?
The Hibernate provider will be loaded dynamically and we don't test validation of `Optional` values.